contributors.txt

```diff
@@ -23,3 +23,4 @@ Olenik Gregor <go@hpsim.de> HPSim
 Ribizel Tobias <mail@upsj.de> Karlsruhe Institute of Technology
 Riemer Lukas <lksriemer@gmail.com> Karlsruhe Institute of Technology
 Tsai Yuhsiang <yhmtsai@gmail.com> National Taiwan University
+Niklas Conen <niklas.conen@student.kit.edu> Karlsruhe Institute of Technology
```

Comment:
We try to keep this list in alphabetical order, last name + first name, so you can move yourself up a bit :) Also if you have a more permanent mail address that you are willing to use, I'd suggest using that instead.